osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -117,14 +126,34 @@ protected override bool OnExiting(Screen next)
             return base.OnExiting(next);
         }
 
+        private void promptExit()
+        {
+            if (confirmExit)
+            {
+                dialog?.Push(new ExitConfirmDialog()
+                {
+                    OnExit = Exit,
+                });
+            }
+            else Exit();
+        }
+
         protected override bool OnKeyDown(InputState state, KeyDownEventArgs args)
         {
+            if (args.Repeat) return false;
+
             if (!args.Repeat && state.Keyboard.ControlPressed && state.Keyboard.ShiftPressed && args.Key == Key.D)

```

Comment:
If you're going to add a seperate if statement for testing if a key press was repeated you can remove this here